api-controllers/DashboardController.js

```diff
@@ -29,13 +29,13 @@ class DashboardController extends FabrikBaseController {
   }
 
   show(req, res) {
-    const managerType = req.instance.plan.manager.name;
-    return req.instance
+    const managerType = req.service.plan.manager.name;
+    return req.service
```

Comment:
Do we need to inject the service in the req object?

Reply:
req.service object is mostly being used for dashboard rendering functions -- as req.instance was being used before. Do you suggest dashboard rendering module should not have external dependencies? This could also be done, but we will still need to use some methods in BoshDirectorClient and DockerService (see [this](https://github.com/cloudfoundry-incubator/service-fabrik-broker/blob/master/broker/lib/fabrik/DirectorManager.js#L196) and [this](https://github.com/cloudfoundry-incubator/service-fabrik-broker/blob/master/broker/lib/fabrik/DockerInstance.js#L60)).
Also I basically took inspiration from [this](https://github.com/cloudfoundry-incubator/service-fabrik-broker/blob/master/api-controllers/ServiceFabrikApiController.js#L938) for using service objects in this way.